public/config-example.json

```diff
@@ -4,6 +4,8 @@
   "hash": "dev",
   "apiUrl": "http://localhost:8180/api",
   "ontologyApiUrl": "https://ontologyURL.org/",
+  "terraUrl": "https://terraURL.org/",
+  "tdrApiUrl": "https://tdrApiUrl.org/",
```

Comment:
We're going to need changes to https://github.com/broadinstitute/terra-helmfile/blob/master/charts/duos/templates/_config.json.tpl for this to work on deployment. Those need to be in before this gets to dev.